.github/workflows/license.yml

```diff
@@ -1,41 +0,0 @@
-# Copyright 2023 DigitalOcean
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-name: License
-
-on:
-  push:
-    branches: [main]
-  pull_request:
-    branches: [main]
-
-jobs:
-  check-file-licenses:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Set up Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: '1.20'
-
-      - name: Download addlicense
-        run: go get -d github.com/google/addlicense
-
-      - name: Build addlicense
-        run: go build -o addlicense github.com/google/addlicense
-
-      - name: Check file licenses
-        run: ./addlicense -c DigitalOcean -l apache -y $(date +"%Y") --check .
```

Comment:
what's the replacement for this code?

Reply:
I accidentally removed this, I can put it back.